manifest_staging/charts/aad-pod-identity/templates/_helper.tpl

```diff
@@ -55,3 +55,14 @@ Common labels.
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 helm.sh/chart: {{ template "aad-pod-identity.chart" . }}
 {{- end -}}
+
+{{/*
+Determine KubeConfig Path
```

Comment:
We can probably just wrap the kubelet-config hostPath and mountPath with the same if statement for `--kubelet-config` instead of using a template since they are not used for < v1.7.0

Reply:
I felt it was more clear and follows DRY to have a template var. Are you wanting me to remove that and put duplicate logical checks in the manifest instead?

Reply:
The templating is good but let's also make sure not to mount the kubelet config path to NMI if the tag is < 1.7.0

Reply:
Also, could you rename it `kubeConfig` to `kubeletConfig`

Reply:
> ...let's also make sure not to mount the kubelet config path to NMI if the tag is < 1.7.0
Not sure I follow; you we're already mounting this config path. In the template, I'm setting this path to `/etc/default/kubelet` if tag is <1.7.0.

> Also, could you rename it from `kubeConfig` to `kubeletConfig`
Sure.

Reply:
The mount path is introduced in the same PR( https://github.com/Azure/aad-pod-identity/pull/842/files#diff-a6197c9515d5b54a66c30c7ffd812d27698d820afbb87671a7b0768cdf651782) so I was wondering if we can also add similar checks where we only mount kubelet config if the tag is < 1.7.0

Reply:
Ok, so you're saying that was a bug that was introduced by that PR and you want to club the fix in this PR?

Reply:
Yea if that's okay with you. If not, I can open a follow-up PR to fix that

Reply:
Ok, I can add. Will also remove the template as it's redundant to check server